capstone/pom.xml

```diff
@@ -75,7 +75,7 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <deploy.projectId>convai-step-2020</deploy.projectId>
+          <deploy.projectId>oalamutu-step-2020</deploy.projectId>
```

Comment:
I think we should change this back to `convai-step-2020` so we don't get weird merge errors later on.

Reply:
addressed in "address nits and add youtube caller support"